index.html

```diff
@@ -1,125 +1,114 @@
+  <head>
+    <meta charset="utf-8" />
+    <meta content="width=device-width, initial-scale=1.0" name="viewport" />
+
+    <title>Miguel Guzman</title>
+    <meta content="" name="description" />
+    <meta content="" name="keywords" />
+
+    <!-- Favicons -->
+    <link href="assets/img/GatoGatito.png" rel="icon" />
+    <link href="assets/img/apple-touch-icon.png" rel="apple-touch-icon" />
+
+    <!-- Google Fonts -->
+    <link
+      href="https://fonts.googleapis.com/css?family=https://fonts.googleapis.com/css?family=Inconsolata:400,500,600,700|Raleway:400,400i,500,500i,600,600i,700,700i"
+      rel="stylesheet"
+    />
+
+    <!-- Vendor CSS Files -->
+    <link href="assets/vendor/aos/aos.css" rel="stylesheet" />
+    <link
+      href="assets/vendor/bootstrap/css/bootstrap.min.css"
+      rel="stylesheet"
+    />
+    <link
+      href="assets/vendor/bootstrap-icons/bootstrap-icons.css"
+      rel="stylesheet"
+    />
+    <link href="assets/vendor/swiper/swiper-bundle.min.css" rel="stylesheet" />
+
+    <!-- Template Main CSS File -->
+    <link href="assets/css/style.css" rel="stylesheet" />
+
+    <!-- Font Awesome Icons File -->
+    <link href="assets/css/all.min.css" rel="stylesheet" />
+
+    <!-- =======================================================
   * Template Name: MyPortfolio - v4.1.0
   * Template URL: https://bootstrapmade.com/myportfolio-bootstrap-portfolio-website-template/
   * Author: BootstrapMade.com
   * License: https://bootstrapmade.com/license/
   ======================================================== -->
-</head>
-
-<body>
-
- 
-  <!-- ======= Navbar ======= -->
-  <div class="collapse navbar-collapse custom-navmenu" id="main-navbar">
-    <div class="container py-2 py-md-3">
-      <div class="row align-items-start">
-
-      <div class="text-md-start col-md-6">
-          <ul class="custom-menu">
-            <a href="about.html">About Me</a> <a>   |   </a>
-            <a href="index.html">Films</a> <a>   |  </a>
-           <a href="contact.html">Contact   </a>
-          </ul>
-        </div>
-      </div>
-
-    </div>
-  </div>
-
-  <nav class="navbar navbar-light custom-navbar">
-    <div class="container">
-      <a class="navbar-brand" href="index.html"></a>
-      <a href="#" class="burger" data-bs-toggle="collapse" data-bs-target="#main-navbar">
-        <span></span>
-      </a>
-    </div>
-  </nav>
-
-
-  <main id="main">
-
-    <!-- ======= Works Section ======= -->
-    <section class="section site-portfolio">
-      <div class="container"> 
-        <div class="row mb-4 align-items-center">
-          <div class="col-md-12 col-lg-6 mb-3 mb-lg-0" data-aos="fade-up">
-            <h2>Miguel Guzmán</h2>
-            <p class="mb-0">Filmmaker</p>
-          </div>
-<!--           <div class="col-md-12 col-lg-6 text-start text-lg-end" data-aos="fade-up" data-aos-delay="100">
+  </head>
+
+  <body>
+    <div w3-include-html="header.html"></div>

```

Comment:
this is the new thing. taken from https://www.w3schools.com/howto/howto_html_include.asp